doc/source/train/user-guides/data-loading-preprocessing.rst

```diff
@@ -276,34 +275,66 @@ At a high level, you can compare these concepts as follows:
      - n/a
      - :meth:`ray.data.Dataset.iter_torch_batches`
 
+Why using Ray Data?
+~~~~~~~~~~~~~~~~~~~
+
+The framework's data utilities work well with small datasets requiring light preprocessing. 
+However, they can become performance bottlenecks when handling large-scale datasets with complex preprocessing logic. 
+Ray Data is designed to address these challenges, providing efficient large-scale data ingestion. 
+
+Specifically, you can benefit from the following features of Ray Data:
+
+**Streaming execution**:
+
+- The preprocessing pipeline will be executed lazily and stream the data batches into training workers.
+- Training can start immediately without significant up-front preprocessing time.
+
+**Automatic data sharding**: 
+
+- The dataset will be automatically sharded across all training workers. 
 
-For more details, see the following sections for each framework.
+**Leverage additional resources for preprocessing**
+
+- Ray Data can utilize all resources in the Ray cluster for preprocessing, not just those on your training nodes. 
+
+For more details, see the following sections for each framework:
 
 .. tab-set::
 
-    .. tab-item:: PyTorch Dataset and DataLoader
+    .. tab-item:: PyTorch
+
+        **Option 1 (with Ray Data):** 
 
-        **Option 1 (with Ray Data):** Convert your PyTorch Dataset to a Ray Dataset and pass it into the Trainer via  ``datasets`` argument.
-        Inside your ``train_loop_per_worker``, you can access the dataset via :meth:`ray.train.get_dataset_shard`.
-        You can convert this to replace the PyTorch DataLoader via :meth:`ray.data.DataIterator.iter_torch_batches`.
+        1. Convert your PyTorch Dataset to a Ray Dataset and 
```

Comment:
nit: 
```suggestion
        1. Convert your PyTorch Dataset to a Ray Dataset.
```

There are some other small typos/formatting errors that I'll review more thoroughly in a follow-up review.